testing/integration/otel_test.go

```diff
@@ -463,22 +426,6 @@ func TestOtelAPMIngestion(t *testing.T) {
 	)
 	require.NoError(t, err, "APM not initialized")
 
-	// wait for otel collector to start
```

Comment:
Is it worth replacing this with a check of the upstream health endpoint?

Reply:
I don't think its worth it at the moment. The work I am doing in #5767 will expose status information over the `elastic-agent status` command again.